src/main/java/org/apache/sling/servlethelpers/MockSlingHttpServletRequest.java

```diff
@@ -516,6 +529,36 @@ public RequestParameterMap getRequestParameterMap() {
         return params;
     }
 
+    public void addRequestParameter(String name, String value) {
```

Comment:
please add a bit of javadocs two this three new addRequestParameter methods

Reply:
Done.  Update to PR is in progress.